docs/standard/terminology.md

```diff
@@ -0,0 +1,69 @@
+# Terminology
+
+In order to use key terms consistently across the standard documentation and OCP communication in general, terminology is managed according to a process that ensures both quality and reactivity.
+
+![Terminology process overview](https://www.lucidchart.com/invitations/accept/2eee45ed-5e62-4636-9a44-24b4414f273f)
+
+## 1. Proposition and validation
+
+When new content is added to the documentation and new concepts are introduced in the standard, new terms may need to be translated.
+
+**Reminder:** a term is a word or group of words that must be used and translated consistently to ensure a good understanding and usage of the content. Synonyms of terms must consequently be banned. Expressions that are not terms are called *generic* expressions.
+
+| #   | Step name   | Description                                                                                          | Tool               |
+| --- | ----------- | ---------------------------------------------------------------------------------------------------- | ------------------ |
+| 1   | Proposition | The writer of the standard writes down a list of words or groups of words they consider being terms. | Github issue       |
+| 2   | Validation  | A terminologist validates that the proposed expressions are actually terms.                          | Github issue       |
```

Comment:
We don't use "terminologist" elsewhere. Is it different from the other roles described?